src/app/public/modules/datepicker/datepicker.component.spec.ts

```diff
@@ -863,6 +863,66 @@ describe('datepicker', () => {
       }));
     });
 
+    describe('Angular form control statuses', function () {
+      it('should set correct statuses on init', fakeAsync(function () {
+        fixture.detectChanges();
+        tick();
+
+        expect(component.dateControl.valid).toBe(true);
+        expect(component.dateControl.pristine).toBe(true);
+        expect(component.dateControl.touched).toBe(false);
+      }));
+
+      it('should set correct statuses on init', fakeAsync(function () {
```

Comment:
Aren't this and the test right above the exact same test? Also, it seems like we would want a test that check the statuses on init when an initial value is given (which is part of what the `firstChange` is for I believe).

Reply:
Yep, must've copied the test in order to test an initial value and didn't change anything. Thanks.

Reply:
Looks good now. Thanks